lib/xcpretty/formatters/formatter.rb

```diff
@@ -27,6 +27,7 @@ def format_linking(file, build_variant, arch);            EMPTY; end
     def format_libtool(library);                              EMPTY; end
     def format_passing_test(suite, test, time);               EMPTY; end
     def format_pending_test(suite, test);                     EMPTY; end
+    def format_measuring_test(suite, test, time);             EMPTY; end
```

Comment:
Unused method argument - `suite`. If it's necessary, use `_` or `_suite` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `test`. If it's necessary, use `_` or `_test` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `time`. If it's necessary, use `_` or `_time` as an argument name to indicate that it won't be used. You can also write as `format_measuring_test(*)` if you want the method to accept any arguments but don't care about them.<br>Use empty lines between defs.<br>Unnecessary spacing detected.